.drone.yml

```diff
@@ -331,9 +393,13 @@ trigger:
   status:
     - success
     - failure
+  exclude:
+    event:
+      - pull_request
 
 depends_on:
   - callisto-timecard-restapi-build
+  - callisto-timecard-restapi-branch-deploy
```

Comment:
slightly confused as to why this is added when pull_request is excluded as the branch deploy is triggered by pull_request

Reply:
We forgot to change this, updated now